src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+function averageAge(ages) {
+  return ages.reduce((sum, x) => sum + x, 0) / ages.length;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century
+    ? men.filter(person => century === Math.ceil(person.died / 100))
+    : men;
```

Comment:
you can avoid one more filter, if you will write condition with ternary operator in
const men = ...